Survey Thank You page styling

technicalex opened the pull request:
Changes to support new styling on Survey Thank You pages.

These changes are compiled to: https://s3.amazonaws.com/actionkit.moveon.org/static/stylesheets/after-actions-thank-you.css

The styling changes interact with templateset changes in the [after-actions-thank-you](https://github.com/MoveOnOrg/ak-template-set/tree/after-actions-thank-you) branch of the ak-template-set repository.

To test, you can apply the `template_set=after-actions-thank-you` parameter to any page, and both the template set changes and the new stylesheet will be applied. You can confirm using Developer tools that the correct stylesheet is being applied.

technicalex commented on src/styles/pages/_survey_thanks.scss:
```diff
@@ -26,7 +26,11 @@ body.survey-pagetype.thanks-page {
 
   .text-info {
-    color: #0098AD; //giraffe has #17a2b8 for .text-info but Brianna says it should be darker teal for this page.
+    // TODO: Aili set this color to #0098AD on Sep 24, 2021, with the following note:
```
I have messaged Aili.

- [ ] Update TODO with Aili's response

technicalex replied:
Aili doesn't have any flags on this change. I will remove the comment block entirely.

strangeways commented on src/styles/pages/_survey_thanks.scss:
```diff
@@ -51,4 +55,96 @@ body.survey-pagetype.thanks-page {
+    }
+  }
+
+  // TODO: Why are some of these colors different from the ones in _social.scss?
```
Regular comment sounds good, since we really should be consistent. As the style guide work proceeds, having a reminder here will helpful.

technicalex commented:
@strangeways, thanks for your review! I have addressed all comments. I compared the `main.css` files generated before and after my latest changes, and saw only inconsequential differences (like lowercase vs. uppercase letters in hex color codes, or `#FFFFFF` instead of `#FFF`). I've attached the files for your examination (with the .txt extension, because GitHub doesn't support .css). The newest file is also uploaded to S3, so it will be pulled whenever you use the `?template_set=after-actions-thank-you` parameter.

Since MoveOn is closed tomorrow, this is the last day of the week, and I don't want to deploy changes right before the long weekend in case of any unexpected issues. Please feel free to deploy these changes when you're back in the office, or I can!

[after-actions-thank-you-after.txt](https://github.com/MoveOnOrg/giraffe/files/9021309/after-actions-thank-you-after.txt)
[after-actions-thank-you-before.txt](https://github.com/MoveOnOrg/giraffe/files/9021310/after-actions-thank-you-before.txt)
